Libraries/LLM/Models.swift

```diff
@@ -110,6 +110,13 @@ public struct ModelConfiguration {
 }
 
 extension ModelConfiguration {
+    public static let smolLM_135M_4bit = ModelConfiguration(
+      id: "mlx-community/SmolLM-135M-Instruct-4bit",
```

Comment:
Also I would try the 16-bit to start just to make sure the bad output is not related to quantization.